tests/core/pyspec/eth2spec/test/fulu/fork_choice/test_on_block.py

```diff
@@ -1 +1,60 @@
-# TODO: add new data availability tests.
+from random import Random
+
+from eth2spec.test.context import (
+    spec_state_test,
+    with_fulu_and_later,
+)
+from eth2spec.test.deneb.fork_choice.test_on_block import get_block_with_blob
+from eth2spec.test.helpers.fork_choice import (
+    BlobData,
+    get_genesis_forkchoice_store_and_block,
+    on_tick_and_append_step,
+    tick_and_add_block_with_data,
+)
+from eth2spec.test.helpers.state import state_transition_and_sign_block
+
+
+@with_fulu_and_later
+@spec_state_test
+def test_simple_blob_data_peerdas(spec, state):
+    """
+    Similar to test_simple_blob_data, but in PeerDAS version that is from FULU onwards.
```

Comment:
Just a nit, Fulu isn't an acronym.
```suggestion
    Similar to test_simple_blob_data, but in PeerDAS version that is from Fulu onwards.
```